host/CMakeLists.txt

```diff
@@ -1,6 +1,38 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
+# fetch branch and last commit information of the OE SDK repository being built
+set(branch_name "Unavailable")
```

Comment:
Please use the existing variables that are already filled out for you in CMake, [`GIT_COMMIT`](https://github.com/Microsoft/openenclave/blob/c5e8a8ff9adeecfdb96509f0738fb301b5daab34/CMakeLists.txt#L36) and [`GIT_BRANCH`](https://github.com/Microsoft/openenclave/blob/c5e8a8ff9adeecfdb96509f0738fb301b5daab34/CMakeLists.txt#L43). You can remove this section entirely.

Thank you for the ping.

Reply:
Sweet!
That works. 
Removed original block an d used GIT_COMMIT and GIT_BRANCH variables instead.